src/screens/ProfileScreen/elements/KarmaLock.tsx

```diff
@@ -14,7 +14,7 @@ export const KarmaLock = (props: {onPressCreatePost: () => void}) => {
       style={{
         backgroundColor: COLORS.gray100,
         borderRadius: 12,
-        paddingVertical: 8,
+        paddingVertical: 6,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The paddingVertical has been reduced from 8 to 6. This is a minor change and should not affect the functionality of the component, but it may slightly alter the visual appearance.
```diff
-        paddingVertical: 8,
+        paddingVertical: 6,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->